iree/hal/cuda/cuda_headers.h

```diff
@@ -8,5 +8,6 @@
 #define IREE_HAL_CUDA_CUDA_HEADERS_H_
 
 #include "cuda.h"  // IWYU pragma: export
+#include "cuda_runtime.h"
```

Comment:
we shouldn't need this and we don't want to add the additional dependency (same with the cmake changes)